LambdaEngine/Source/Rendering/LineRenderer.cpp

```diff
@@ -358,9 +358,13 @@ namespace LambdaEngine
 		uint32 modFrameIndex,
 		uint32 backBufferIndex,
 		CommandList** ppFirstExecutionStage,
-		CommandList** ppSecondaryExecutionStage)
+		CommandList** ppSecondaryExecutionStage,
+		bool Sleeping)
```

Comment:
Functions use pascalcase. Parameters and local variables use camelcase. `bool Sleeping` -> `bool sleeping`